frontend/pages/hosts/details/cards/Software/SelfService/SelfServiceItem/SelfServiceItem.tsx

```diff
@@ -20,11 +20,28 @@ import { IStatusDisplayConfig } from "../../InstallStatusCell/InstallStatusCell"
 
 const baseClass = "self-service-item";
 
+// TODO: This is all todo, I just had to remove installed key
+// since the type installed was giving me typing errors for SoftwareInstallStatus on my branch
 const STATUS_CONFIG: Record<SoftwareInstallStatus, IStatusDisplayConfig> = {
-  installed: {
+  verified: {
     iconName: "success",
-    displayText: "Installed",
-    tooltip: ({ lastInstalledAt }) => (
+    displayText: "Installed", // TODO: Update tests to change this to Verified
+    tooltip: (
+      { lastInstalledAt } // TODO: confirm all tooltips
+    ) => (
+      <>
+        Software installed successfully ({dateAgo(lastInstalledAt as string)}).
+        Currently, if the software is uninstalled, the &quot;Installed&quot;
+        status won&apos;t be updated.
+      </>
+    ),
+  },
+  verifying: {
+    iconName: "success-outline",
+    displayText: "Verifying", // TODO
```

Comment:
Can you leave a bit more info on what's left TODO? Hard for me to tell if it's intentional or not otherwise

Reply:
same